src/main/java/sandbox/SqlSandboxUtils.java

```diff
@@ -51,7 +51,7 @@ public static SQLiteConnectionPoolDataSource getDataSource(String databaseName,
             }
             Class.forName("org.sqlite.JDBC");
             SQLiteConnectionPoolDataSource dataSource = new SQLiteConnectionPoolDataSource();
-            dataSource.setUrl("jdbc:sqlite:" + databasePath + "/" + databaseName + ".db");
+            dataSource.setUrl("jdbc:sqlite:" + databasePath + "/" + databaseName + ".db?journal_mode=MEMORY");
```

Comment:
Switching our journaling mode to `memory`. Per SQLite docs:

The MEMORY journaling mode stores the rollback journal in volatile RAM. This saves disk I/O but at the expense of database safety and integrity. If the application using SQLite crashes in the middle of a transaction when the MEMORY journaling mode is set, then the database file will very likely go corrupt.

Since we're getting killed on I/O but have memory to spare and don't care about database corruption in the event of a crash, I think this is the way to go